src/App.tsx

```diff
@@ -1,223 +1,272 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import classNames from 'classnames';
 import { AuthContext } from './components/Auth/AuthContext';
+import {
+  addTodo,
+  changeTodo,
+  getTodos,
+  removeTodo,
+} from './api/todos';
+import { Todo } from './types/Todo';
+import { Types } from './types/Types';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
+import { NewTodo } from './components/NewTodo/NewTodo';
+import { ErrorNote } from './components/ErrorNote/ErrorNote';
+
+const FILTERS: Types = {
+  all: 'all',
+  completed: 'completed',
+  active: 'active',
+};
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState(FILTERS.all);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [errorStatus, setErrorStatus] = useState(false);
+  const [isAdding, setIsAdding] = useState(false);
+  const [isRemoving, setIsRemoving] = useState<number[]>([]);
```

Comment:
it's not clear, why isRemoving variable, that probably should store some boolean value, stores array of numbers instead
check the variable name

Reply:
done